source/_docs/configuration/secrets.markdown

```diff
@@ -35,6 +35,7 @@ The `secrets.yaml` file contains the corresponding password assigned to the iden
 ```yaml
 http_password: YOUR_PASSWORD
 ```
+If password contains the hash character, you have to embed it in double quotes to avoid it being treated as a comment. `http_password: "YOUR_PASSWORD_WITH_#"
```

Comment:
That's YAML and already covered in the documentation.

Reply:
Well, if that logic is applied generally, most of the doc would be superfluous ;) It's common mistake, and not mentioned anywhere in the docs that are usually read y inexperienced users. Maybe it should rather be mentined as a "Tip"?

Reply:
Ah, sorry, then I misunderstood your first comment. If it's covered in the config then it's probably OK. Even though I don't think it's bad to ad hints and tricks, it's hard to "overdocument."